docs/articles/baseplatform/booting-qemu-image.rst

```diff
@@ -39,7 +42,56 @@ following command:
       -no-reboot \
       -soundhw ac97 \
       -m 512 \
-      --append "vga=0 uvesafb.mode_option=1280×720-32 root=/dev/hda console=ttyS0 rw mem=512M oprofile.timer=1 " -serial stdio
+      -append "vga=0 uvesafb.mode_option=1280×720-32 root=/dev/hda console=ttyS0 rw mem=512M oprofile.timer=1 " -serial stdio
+
+Static IP address configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A static IP address facilitates development of automation and testing scripts.
+The next commands assumes target ip address is ``192.168.7.2`` and host machine
```

Comment:
Can we be more specific when we say "target" and talk about a QEMU target. As a developer, this is confusing, so I assume that a regular user would also be confused by it?